Library/SpecUI/CommonUI/CWin/cwinBase.asm

```diff
@@ -1400,13 +1399,13 @@ continue:
 	mov	ax, HINT_AVOID_MENU_BAR
 	call	ObjVarFindData
 	jc	done
-if EXTENDIBLE_SYSTEM_TRAY
+;if EXTENDIBLE_SYSTEM_TRAY ; - this flag is dead. Taskbar also means Systray.

```

Comment:
Please cleanup the condition here....

Reply:
It's a little bit unclear if this code is still needed, but I removed the condition and left it in, adding a comment.